components/DatePicker.js

```diff
@@ -0,0 +1,236 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import Flatpickr from 'flatpickr';
+import ReactDOM from 'react-dom';
+
+class DatePicker extends Component {
+  state = {
+    cal: {}
+  }
+
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    short: PropTypes.bool,
+    datePickerType: PropTypes.string,
+    dateFormat: PropTypes.string
+  };
+
+  static defaultProps = {
+    short: false,
+    dateFormat: 'm/d/Y'
+  };
+
+  componentDidMount() {
+    if (this.props.datePickerType === 'single' || this.props.datePickerType === 'range') {
+      // Weekdays shorthand for english locale
+      Flatpickr.l10ns.en.weekdays.shorthand.forEach((day, index) => {
```

Comment:
Given `Flatpickr.l10ns` is a system-wide setting, we shouldn’t need to tweak it every time a component mounts. We can do it outside the `DatePicker ` class definition.